releasenotes/jmri5.5.3.shtml

```diff
@@ -108,7 +108,9 @@ that can arise from skipping these versions.
 <a id="problems" name="problems"></a>
 <h3>New problems with this release</h3>
 <ul>
-    <li>None yet</li>
+    <li>TurnoutStatePersistence.py was not correctly updated to use SLF4J, line 127 should be
+    <br><code>    log = LoggerFactory.getLogger("jmri.jmrit.jython.exec.TurnoutStatePersistence.LoadTurnoutState")</code>
```

Comment:
Is this fixed in JMRI 5.5.4?

Reply:
Yep, see https://github.com/JMRI/JMRI/pull/12263